lib/loader/index.js

```diff
@@ -74,6 +80,12 @@ function instantiate(module, imports) {
     return parts.join("") + String.fromCharCode.apply(String, U16.subarray(dataOffset, dataOffset + dataRemain));
   }
 
+  /** Frees a string in the module's memory by its pointer. */
+  function freeString(ptr) {
+    checkMem();
```

Comment:
Doesn't need to checkMem here. That's only necessary when accessing one of the views I32, F32 etc.

Reply:
Yeah you're right. I'll change this